tests/test_check_ghostty_zig_workflows.py

```
@@ -0,0 +1,332 @@
+            runs-on: ubuntu-latest
+            steps:
+              - name: Execute
+                run: |
+                  echo preparing
+                  ./scripts/install-zig-ci.sh
+        """
+    )
+
+    assert failures == [
+        "fixture.yml:10: build executes scripts/install-zig-ci.sh "
+        "before Ghostty submodule init"
```

Ruff (0.16.0)

[warning] Unparenthesized implicit string concatenation in collection

Did you forget a comma?

(ISC004)